### PR TITLE
Add auth

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Add Authentication
User can signup with details
Then users can login successfully

### CODE REVIEW

File: src/pages/Auth/login/login.jsx [hunk 1]
```diff
@@ -1,26 +1,57 @@
...
+    const body = {
+    email : email,
+    password : password,
+ }
+try {
+  const response = await axios.post("/api/auth/login",body)
+  response.data.encodedToken? navigate("/") : alert("login failed")
+
+} catch (error) {
+  console.error("error")
```
Comment (harshitpaliwal95) [inline src/pages/Auth/login/login.jsx:23]: use error.message

**Review by harshitpaliwal95 — COMMENTED** [review]
every thing looksgood

**Review by ckahub — APPROVED** [review]
code has good structure
code is readable
code is neat and clean

**Comment by k-rahul21** [comment]
Your code looks clean and good, nicely implemented.

**Review by k-rahul21 — APPROVED** [review]

**Pull request merged**